src/components/OntologyBrowser/store.ts

```diff
@@ -10,6 +12,15 @@ export type PhenotypeSource = {
   filter_by_term: any;
 };
 
+export type TreeNodeLight = {
+  key: string;
+  title: React.ReactElement | string;
+  children: TreeNode[];
+};
+
```

Comment:
I'm not convinced that a new type is the way to go here. That will make the Node incompatible with other process and there is not that much difference.

I think it would be better to have a type constructor that return a built 'light' object. (set the default)

And please put the type definition in the model file.